makefile

```diff
@@ -73,7 +73,7 @@ parser/machine.go: parser/machine.go.rl common/common.rl
 
 parser/machine.go:
 	$(RAGEL) -Z -G2 -e -o $@ $<
-	@sed -i '/^\/\/line/d' $@
+	@sed -i '' '/^\/\/line/d' $@
```

Comment:
A better approach would be to get rid of `sed` so as to avoid having to take into account the differences between OS X's sed and GNU sed (things like the `-i` flag...).

Thus, I plan to substitute sed with some other approach that works consistently across different platforms.